lib/android/app/src/main/java/com/reactnativenavigation/react/NavigationModule.java

```diff
@@ -72,6 +73,27 @@ public String getName() {
         return NAME;
     }
 
+    @ReactMethod
+    public void getLaunchArgs(String commandId,Promise promise) {
+        NavigationActivity activity = activity();
+        if (activity != null ) {
+            Intent intent = activity.getIntent();
+            if (intent != null) {
+                Bundle launchArgs = intent.getBundleExtra("launchArgs");
+                if(launchArgs != null){
+                    WritableMap writableMap = Arguments.fromBundle(launchArgs);
+                    promise.resolve(writableMap);
+                }else{
```

Comment:
Seems like all of these else statements are identical and can be removed.
If you're unable to obtain the `launchArgs` - return `Arguments.createMap()`. Don't need do return it from every nested conditional.

Reply:
refactored